### DISCUSSION TITLE
Here's my test for Galaxy Brain #2 and anything that we know

### DISCUSSION THREAD

**Discussion by LumineZhao** [Schweinepriester/github-profile-achievements#118] [Q&A] [answered]
This is a discussion created by 3rd party user. This is going to be answered by my partner.

**Comment by SunsetMkt** [comment, marked as answer]
This is an answer.

**Comment by LumineZhao** [comment]
It seems that Galaxy Brain is not working properly as we expected since she has answered 6 answers (marked) and haven't got her achievement.

**Reply by Schweinepriester** [reply]
Are any of those 6 marked as the answer by herself?

Because recently there was a bug if you marked any answer yourself, you aren't able to get it.

https://github.com/Schweinepriester/github-profile-achievements/discussions/58#discussioncomment-3025210

**Reply by LumineZhao** [reply]
> Are any of those 6 marked as the answer by herself?
> 
> Because recently there was a bug if you marked any answer yourself, you aren't able to get it.
> 
> [#58 (reply in thread)](https://github.com/Schweinepriester/github-profile-achievements/discussions/58#discussioncomment-3025210)

Yes, you are right. Two of them are marked by herself during our test.

She has got her achievement after un-marking them and re-making this answer again.

It seems that there's limitation on 2rd party repos (my repo, my discussion, her answer).
